wso2-streaming-integrator-mixin/dashboards/Siddhi_server.json

```diff
@@ -1082,7 +1082,7 @@
           {
             "matcher": {
               "id": "byName",
-              "options": "jvm_memory_heap_max{instance=\"localhost:9005\",job=\"PrometheusReporter\"}"
+              "options": "jvm_memory_heap_max{instance=\"$instance\"}"
```

Comment:
Just a question, this dashboard has a mix of job + instance, and instance only. When is the distinction made?